public/app/features/plugins/admin/components/PluginDetailsBody.tsx

```diff
@@ -18,10 +19,29 @@ type Props = {
   pageId: string;
 };
 
+type Cell<T extends keyof Permission = keyof Permission> = CellProps<Permission, Permission[T]>;
+
 export function PluginDetailsBody({ plugin, queryParams, pageId }: Props): JSX.Element {
   const styles = useStyles2(getStyles);
   const { value: pluginConfig } = usePluginConfig(plugin);
 
+  //@ts-expect-error
+  const columns: Array<Column<Permission>> = useMemo(
+    () => [
+      {
+        id: 'action',
+        header: 'Action',
+        cell: ({ cell: { value } }): Cell<'action'> => value,
```

Comment:
Shouldn't this be `cell: ({ cell: { value } }: Cell<'action'>) => value,`?

Reply:
Indeed. And this also fixes the typescript problem that is ignored in this variable declaration.

Reply:
Damn, this is probably it. Spent a while yesterday trying to figure out the cause of that type error 😅

Reply:
Great eye @jackw 

We spent some time trying to figure this one out and because of the data structures needed for `InteractiveTable` we missed this. This also helps with the ugly `//ts-expect-error`

Thank you for your review!